README.md

```diff
@@ -157,12 +157,12 @@ A more complete overview of our stack is available in
 
 ## Vulnerability disclosure
 
-Forem is the open source software which powers [DEV](https://dev.to).
+Forem is the open-source software which powers [DEV](https://dev.to).
 
 We welcome security research on DEV under the terms of our
 [vulnerability disclosure policy](https://dev.to/security).
 
-## Acknowledgments
+## Acknowledgements
```

Comment:
I think this is a case of American English (what we have currently) versus British English (what is being suggested in this PR). Since we _tend_ to use the American English elsewhere, I think we should stick with that.
```suggestion
## Acknowledgments
```